app/controllers/concerns/mfa_setup_concern.rb

```diff
@@ -82,7 +82,7 @@ def show_skip_additional_mfa_link?
   end
 
   def check_if_possible_piv_user
-    if current_user.has_gov_or_mil_email? && current_user.piv_cac_recommended_dismissed_at.nil?
+    if current_user.gov_or_mil_email? && current_user.piv_cac_recommended_dismissed_at.nil?
```

Comment:
One difference I notice in an example like this is dropping the `has_` makes it less clear whether we expect a user to have one vs. multiple emails to checking. I wonder if prefixing `any_` instead could help reinforce that?

```suggestion
    if current_user.any_gov_or_mil_email? && current_user.piv_cac_recommended_dismissed_at.nil?
```

Reply:
I agree that it makes it less clear, but to me this is an argument that "**avoid** predicate methods" is a recommended best practice for developers to keep in mind, not something that's a mandate. I don't think `any_gov_or_mil_email?` is clearer than `has_gov_or_mil_email?`.

Reply:
I think I'll undo this and remove `has_` as a prefix that is blocked, because it seems to help clarify in our usages